components/brave_wallet_ui/common/hardware/ledgerjs/filecoin_ledger_keyring.test.ts

```diff
@@ -31,12 +31,6 @@ const createFilecoinKeyring = () => {
   return ledgerHardwareKeyring
 }
 
-test('Extracting BLS public key', () => {
- const privateKey = '6a4b3d3f3ccb3676e34e16bc07a9371dede3a037def6114e79e51705f823723f'
- return expect(extractPublicKeyForBLS(privateKey)).toStrictEqual(
-   '623537373466336438353436643365373937363533613534323365666661376162303664346364333538373639376433363437373938643966653733393136376562656166316566303533663935376137363738656534646530653332613833')
```

Comment:
it seems like this test was removed without an equivalent replacement?

Reply:
it was only for typescript library, now we have https://github.com/brave/brave-core/pull/11866/files#diff-6075e572beb7aed954e69f078a852eeb768b9b16f443369c9f48be97b76d191dL29  this one to test Rust lib calls

Reply:
can we add the equivalent check with a call to `fil_private_key_public_key` somewhere around line 34?

Reply:
done